### PR TITLE
Add exported identifier to ExportSpecifier for esprima

### PR DESCRIPTION
**Pull request by adamjmcgrath**
See #271

### CODE REVIEW

**Comment by michaelficarra** [comment]
That is not where the test belongs. Please move the tests to their own files.

**Comment by adamjmcgrath** [comment]
No worries - have added a separate test case

File: escodegen.js [hunk 1]
```diff
@@ -2262,10 +2262,10 @@
         },
 
         ExportSpecifier: function (expr, precedence, flags) {
-            var exported = (expr.id || expr.imported).name;
-            var result = [ exported ];
+            var name = (expr.id || expr.imported || expr.exported).name;
```
Comment (michaelficarra) [inline escodegen.js]: I think `imported` should be removed. Does that break any tests?

**Comment by adamjmcgrath** [comment]
My guess is this would break https://github.com/estools/escodegen/blob/master/test/compare-esprima2/import-with-default.js#L5 because `ExportSpecifier` is used by `ImportSpecifier`.

**Comment by michaelficarra** [comment]
Oh, really? That's a bit gross. Okay, I will look into how that could be cleaned up. Merging for now. Thanks.

**Pull request merged**